src/averageAges.js

```diff
@@ -20,6 +20,15 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  const men = century
+    ? people.filter(person => Math.ceil(person.died / 100) === century
+      && person['sex'] === 'm')
+    : people.filter(person => person['sex'] === 'm');
+
+  return men
```

Comment:
u can move logic of looking for average to separate functions to prevent duplicating